tasks/main.yml

```diff
@@ -5,22 +5,43 @@
 - include: setup-Debian.yml
   when: ansible_os_family == 'Debian'
 
+- include: elasticdump.yml
```

Comment:
should we call this file elasticdump-debian.yml to conform to this repo's other includes, and leave room for someone to contribute a redhat version later?